util_scripts/dev-api-encoded-secret.sh

```diff
@@ -41,10 +42,15 @@ echo "---------------------------------------------------------------"
 # curl -X GET --location  "$COMMUNITY_HOST/secure/staff/staffIdentifier/2500012436" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 
 
-curl -X GET --location  "$COMMUNITY_HOST/secure/staff/username/lucybennett" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
+# curl -X GET --location  "$COMMUNITY_HOST/secure/staff/username/lucybennett" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
+curl -X GET --location  "$COMMUNITY_HOST/secure/staff/username/timharrison" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 
 # curl -X GET --location  "$COMMUNITY_HOST/secure/offenders/crn/X356288/allOffenderManagers" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 # curl -X GET --location  "$COMMUNITY_HOST/secure/offenders/crn/X356288/contact-summary" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 # curl -X GET --location  "$COMMUNITY_HOST/secure/staff/staffIdentifier/2500002049/managedOffenders?current=true" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 
+# Prison register
+curl -v --location  "$PRISON_REGISTER_HOST/prisons/id/PBI" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
```

Comment:
Not sure what this script does.. we dont use Prison Register API in this service..

Reply:
Its just an ad-hoc utility script I use to query APIs - like a manual Postman, for my convenience only.